### ISSUE TITLE
folder in file list has file icon

### ISSUE THREAD

**Issue by dhebbeker** [doxygen/doxygen#7652]
I have the following scenario:
```
C:.
│   Doxyfile
│
└───a
    │   dummy.hpp
    │   README.md
    │
    └───aa
        │   README.md
        │
        └───aaa
                README.md
```

In each [`README.md`](https://github.com/doxygen/doxygen/files/4358706/README.md.txt) I have a `\dir .` documentation for that directory. Generating the documentation for that (`EXTRACT_ALL`, `RECURSIVE`; see [`Doxyfile`](https://github.com/doxygen/doxygen/files/4358695/Doxyfile.txt)).

In the resulting file page the directoy `aaa` has a *file icon*.

![grafik](https://user-images.githubusercontent.com/1594340/77152675-b3108900-6a98-11ea-8f17-add306571ddf.png)

I consider this a bug.

---
Setup:

- doxygen version 1.8.17 (b5fa3cd1c6e6240e20d3b80a70e3f04040b32021*)
- Windows 10 64bit
- [`Doxyfile`](https://github.com/doxygen/doxygen/files/4358695/Doxyfile.txt)

**Comment by albert-github** [comment]
I can see the issue, but it took me some time to reproduce it as at first I missed the `dummy.hpp`, even though all relevant information is present, for the future:
- please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent.

Full example: [example.tar.gz](https://github.com/doxygen/doxygen/files/4358947/example.tar.gz)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7654

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.18.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
